ui/src/types/pdfjs-viewer.ts

```diff
@@ -1,4 +1,25 @@
-import { PDFDocumentProxy, PDFPageProxy, PDFPageViewport } from "pdfjs-dist";
+import { PDFDocumentProxy, PDFPageProxy } from "pdfjs-dist/types/display/api";
+
+interface PDFPageViewportOptions {
+  viewBox: any;
+  scale: number;
+  rotation: number;
+  offsetX: number;
+  offsetY: number;
+  dontFlip: boolean;
+}
+
+interface PDFPageViewport {
+  width: number;
+  height: number;
+  scale: number;
+  transforms: number[];
+
+  clone(options: PDFPageViewportOptions): PDFPageViewport;
+  convertToViewportPoint(x: number, y: number): number[]; // [x, y]
+  convertToViewportRectangle(rect: number[]): number[]; // [x1, y1, x2, y2]
+  convertToPdfPoint(x: number, y: number): number[]; // [x, y]
+}
```

Comment:
nit pick: can these two declarations be moved below the block comment below? The block comment below provides exposition describing why the file exists, so it might benefit from continuing to be at the top of the file.

Reply:
Great, thank you!